### DISCUSSION TITLE
msql 有特殊字段如from 使用结构体使用insert报错

### DISCUSSION THREAD

**Discussion by lopo1** [rbatis/rbatis#529] [General]
报错如下：
NFO rbatis::plugin::intercept_log:[rbatis] [646562649806475270] <= 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'from,to,symbol,decimals,remark,value,status,state,push_number,created_at,time) V' at line 1

**Comment by zhuxiujia** [comment]
> 报错如下： NFO rbatis::plugin::intercept_log:[rbatis] [646562649806475270] <= 1064 (42000): You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'from,to,symbol,decimals,remark,value,status,state,push_number,created_at,time) V' at line 1

数据库限制，建议from加上引号
